src/angle_builder/angle.py

```diff
@@ -330,6 +332,223 @@ def _create_macos_dylibs(self, output_artifact_mode: str) -> list:
 
         return [libEGL_dylib_path, libGLESv2_dylib_path]
 
+    def _create_iphoneos_frameworks(self, output_artifact_mode: str) -> list:
+
+        output_extension = (
+            "xcframework"
+            if output_artifact_mode == "iphoneall-universal"
+            else "framework"
+        )
+
+        libEGL_output_path = os.path.join(
+            self.angle_path,
+            "out",
+            output_artifact_mode,
+            f"libEGL.{output_extension}",
+        )
+        libGLESv2_output_path = os.path.join(
+            self.angle_path,
+            "out",
+            output_artifact_mode,
+            f"libGLESv2.{output_extension}",
+        )
+
+        self._logger.info(
+            "Creating iphoneos frameworks for branch %s and build target %s",
+            self.branch,
+            output_artifact_mode,
+        )
+
+        if output_artifact_mode == "iphoneall-universal":
+            # Ensure the fake all-universal folder exists
+            iphone_universal_folder = os.path.join(
+                self.angle_path, "out", "iphoneos-arm64", "iphoneall-universal"
+            )
+            # Clean it up if it exists
+            if os.path.exists(iphone_universal_folder):
+                shutil.rmtree(iphone_universal_folder)
```

Comment:
Minor suggestion: using `ignore_errors=True` flag would save a line here and in the other rmtree() call. Even though it would also ignore (not raise) other errors like `PermissionError` so it's not fully equivalent to the line above